.prettierrc.cjs

```diff
@@ -4,4 +4,13 @@ module.exports = {
   semi: true,
   singleQuote: true,
   printWidth: 100,
+  overrides: [
```

Comment:
@dannywillems FYI the config in `.prettierrc.md.cjs` was not used by my IDE, so I copied it into this file.  It should do the same thing, and maybe we can also delete `.prettierrc.md.cjs`.